src/Makefile.am

```diff
@@ -677,13 +677,13 @@ FORCE:
 # Otherwise, set it from the contents of .git_version.
 
 ceph_ver.h: .git_version
-	if [ -n "$$NO_VERSION" ] ; then \
-	    if [ ! -f ./ceph_ver.h ] ; then \
-	        $(srcdir)/make_version -n ./ceph_ver.h ; \
-	    fi; \
-        else \
-	    $(srcdir)/make_version $(srcdir)/.git_version ./ceph_ver.h ; \
-	fi
+if NO_VERSION
```

Comment:
I was suggesting something like
<pre>
if NO_VERSION
export NO_VERSION=
else
export NO_VERSION=true
endif
</pre>
why would we need something more ?